### DISCUSSION TITLE
Comment-out code block

### DISCUSSION THREAD

**Discussion by mbakker7** [jupyterlab/jupyterlab#14577] [Q&A] [answered]
Anybody know how to comment-out a code block in JupyterLab 4?
It used to be: select code and use CTRL + /
Doesn't seem to work when running Jupyter Lab 4 on Safari. Is there another magical key-combination?

**Comment by ccalvocm** [comment, marked as answer]
Greetings @mbakker7 

Yes, there is another way to comment-out a code block in JupyterLab 4. Here's how you can do it:

1. Select the code block that you want to comment-out.
2. Press the Cmd key and the / key at the same time. This will comment-out the selected code block.

Alternatively, you can also use the following key combination:

1. Select the code block that you want to comment-out.
2. Press the Ctrl key and the Shift key and the C key at the same time. This will comment-out the selected code block.

These key combinations should work in JupyterLab 4 on Safari. Let me know if you have any other questions or concerns.

**Reply by mbakker7** [reply]
Cmd + / works like a charm. Thanks!
(But Ctrl + Shift + C does not work on my Mac. But that is a less convenient key combo for me anyway.)

**Reply by dinohsu1019** [reply]
In Windows 11, neither works.
1. Ctrl + / works occasionally, but more often it becomes ''' (3 apostrophes)
2. Ctrl + Shift + C: I see search box pops up, nothing to do with commenting.
I have been frustrated by this problem (Ctrl + / not working) for 9 months now.
In the meantime, there's no replacement within selection, which is also missing but very basic edit function.
